LoRaEngine/LoraKeysManagerFacade/DeviceGetter.cs

```diff
@@ -62,6 +62,7 @@ public DeviceGetter(RegistryManager registryManager, ILoRaDeviceCacheStore cache
             try
             {
                 DevNonce? devNonce = ushort.TryParse(rawDevNonce, NumberStyles.None, CultureInfo.InvariantCulture, out var d) ? new DevNonce(d) : null;
+                DevAddr? devAddr = Hexadecimal.TryParse(devAddrString, out uint rawDevAddr) ? new DevAddr(rawDevAddr) : null;
```

Comment:
shouldn't we encapsulate DevAddr parsing logic into the DevAddr?

Reply:
Ah yes, apologies, had the change in a different commit and did not test/push it yet. Will do that

Reply:
Addressed and tested with 0ccc5f6d6bc95b7f66651e17c233478a726324ef